packages/react-aria-components/src/Meter.tsx

```diff
@@ -58,7 +58,12 @@ function Meter(props: MeterProps, ref: ForwardedRef<HTMLDivElement>) {
   });
 
   return (
-    <div {...meterProps} {...renderProps} ref={ref} slot={props.slot}>
+    <div 
+      {...meterProps} 
+      {...renderProps} 
+      ref={ref} 
+      slot={props.slot}
+      data-valuetext={meterProps['aria-valuetext']}>
```

Comment:
hmm I wonder what people would use this one for. It's not a boolean state so not sure? Also looks like ProgressBar has the same one.

Reply:
yeah i noticed that too. figured it wouldn't hurt to have it so i made it a data attribute but not opposed to changing it